### DISCUSSION TITLE
Duda sobre el hackaton

### DISCUSSION THREAD

**Discussion by 11gemii** [Launch-X-Latam/MisionFrontEnd#390] [General]
Veo que todos están formando equipos, es un evento en línea? y yo tengo muy poco aprendiendo sobre programación se necesita un nivel muy avanzado o como principiante se puede participar?

**Comment by iam-leo** [comment]
Al finalizar este programa habrá una hackaton los días 17 y 18 de diciembre. La idea es aplicar todo lo aprendido aquí, no necesariamente debes tener un nivel alto de conocimientos. Si ya estás dentro de las misiones frontend y backend, y llevas las prácticas al corriente entonces si busca formar grupo, si no, te recomiendo que soluciones primero ingresar a las misiones, luego terminar las prácticas y ya por último busca formar grupo ya que la hackaton es en diciembre hay tiempo para ocuparse de eso luego, pero si asegúrate de tener todas las prácticas en tu Github que te servirá para mostrarte al mundo y demostrar de lo que sabes hacer!
